apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -358,10 +362,12 @@ class NewRewardClass extends React.Component {
       />
       <FormField
         label="End date"
+        width={`calc(50% - ${GU}px)`}
```

Comment:
This is a nice improvement. But in what way is it related to the issue at hand?

Reply:
It was really low hanging fruit for a big visual improvement on mobile screens. Why wouldn't I fix it?

Reply:
Fair enough. It's just confusing for a reviewer to understand where you were coming from, so I think it's a good practice to provide a small explanation in the PR to help review it. Because as a reviewer you're out of context and it's hard to tell the really meaningful changes from other improvements, and the less changes the better.

Also bear in mind that sometimes other people notice the same low-hanging fruit as you and fix it in another PR that gets merged more or less at the same time as yours, which could cause avoidable conflicts. So the best practice is to create a dedicated PR, especially if the change is big, or at least check that there is no other pending PR that solved it already.

Reply:
Understood. I'll try to separate those sorts of changes into their own PRs. Its been hard to avoid fixing small issues as they come up in unrelated issues but you make a valid point :)

Reply:
Something that helps me with things like this: [`hub`](https://github.com/github/hub). Here's my basic workflow:

* When I stage changes, I always use `git add -p`
* I stage only things that are relevant to the current effort
* Commit them
* `git checkout dev && git checkout -b new-branch-name-here`
* Then I `git add -p` again, stage & commit the unrelated changes – make the commit message the same thing you want to say in your PR title + description
* `git push`
* `git pull-request`

Then, with a few commands that become pretty automatic once you do them enough, you can quickly push all those good cleanup changes to their own dedicated PR.